logistics/scripts/Invoke-SdkGen.ps1

```diff
@@ -85,7 +85,7 @@ function Invoke-SdkGen {
                 $script:result += Invoke-Task "Invoke-RebuildSolution" { Invoke-RebuildSolution $buildConfiguration "minimal"  $sdkGenSolution }
                 $script:result += Invoke-Task -name "Start-TestHarness" -task { Start-TestHarness $apiUrl $configurationFile $environmentFilePath $null "EdFiOdsApiSdk" }
                 $sdkCliVersion = Get-ValueOrDefault $teamCityParameters['SdkCliVersion'] '7.2.0'
-                $arguments = @("-v",$sdkCliVersion,"--core-only","-j","""$javaPath""")
+                $arguments = @("-v",$sdkCliVersion,"-c","-i","-j","""$javaPath""")
```

Comment:
This seems to have been mistakenly set to `--core-only` in #1069, so I reverted the change.